cleverpush/src/main/java/com/cleverpush/CleverPush.java

```diff
@@ -2514,6 +2514,19 @@ public void showTopicsDialog(Activity dialogActivity, TopicsDialogListener topic
                 alert.setOnShowListener(dialog -> {
                     Logger.d(LOG_TAG, "showTopicsDialog activity: " + dialogActivity.getClass().getCanonicalName());
                     showingTopicsDialog = true;
+                    int currentNightMode = dialogActivity.getResources().getConfiguration().uiMode
+                            & Configuration.UI_MODE_NIGHT_MASK;
+                    switch (currentNightMode) {
+                        case Configuration.UI_MODE_NIGHT_NO:
+                            alert.getButton(AlertDialog.BUTTON_POSITIVE).setTextColor(dialogActivity.getResources().getColor(R.color.cp_black));
```

Comment:
I am not sure if we should hard-code `color.cp_black` and `color.cp_bannerBackground`
What if the customer has set a custom style for his activity? We would then override this style. We need to find a solution where we do not need to hard-code any black/white colors in our code.

Reply:
We are not specifying any style for topic dialog. it will take the default style if device's dark mode is on than it will automatically change the popup background color to black and text color to white. vise versa for light mode. because of that it's hard coded. color.cp_black and color.cp_bannerBackground are taken from sdk's color.xml